vendor/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -200,7 +200,7 @@ def get_report_data_from_plist(plist, skip_report_handler=None):
             report_events.append({'line': event.location['line'],
                                   'col':  event.location['col'],
                                   'file': event.location['file'],
-                                  'msg':  event.message,
+                                  'msg':  event.message.replace('\n', '<br>'),
```

Comment:
Replace new line characters in the JavaScript not in the Python code:
https://github.com/Ericsson/codechecker/blob/8cc87e56be5fbc8ce79c4d7c57a779078964dc86/vendor/plist_to_html/plist_to_html/dist/js/bugviewer.js#L167
```js
enumeration.innerHTML = bugEvent.step.replace(/(?:\r\n|\r|\n)/g, '<br>');
```